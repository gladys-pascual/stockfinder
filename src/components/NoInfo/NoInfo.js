import { Link } from "react-router-dom";
import "./NoInfo.scss";

const NoInfo = () => {
  return (
    <div className="no-info">
      <div className="heart-broken-wrapper">
        <i className="fas fa-heart-broken heart-broken">
          <span className="sr-only">Heart broken emoji</span>
        </i>
      </div>
      <h1 className="no-info-heading">
        Unfortunately, information about this stock is not available.
      </h1>
      <h1 className="no-info-heading">
        Please{" "}
        <Link exact={true} to="/">
          search
        </Link>{" "}
        for another stock.
      </h1>
    </div>
  );
};

export default NoInfo;
