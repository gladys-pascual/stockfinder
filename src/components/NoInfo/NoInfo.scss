@import "../../assets/variables.scss";

.no-info {
  padding: 20px 80px;
  .no-info-heading {
    font-size: 2em;
    font-weight: 200;
    line-height: 1.8em;
    font-family: "Roboto", sans-serif;
    letter-spacing: 4px;
    text-align: center;
    @media screen and (max-width: 700px) {
      font-size: 1.5em;
    }
    a {
      text-decoration: none;
      color: black;
      font-weight: 500;
    }
    a:hover {
      cursor: pointer;
      color: darkgrey;
    }
  }
  .heart-broken-wrapper {
    display: flex;
    justify-content: center;
    .heart-broken {
      font-size: 80px;
      color: $theme-color;
    }
  }
}
