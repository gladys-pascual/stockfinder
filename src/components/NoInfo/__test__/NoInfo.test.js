import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import NoInfo from "../NoInfo";

test("render NoInfo component as expected", () => {
  const { container } = render(
    <BrowserRouter>
      <NoInfo />
    </BrowserRouter>
  );

  expect(container).toMatchSnapshot();
});
