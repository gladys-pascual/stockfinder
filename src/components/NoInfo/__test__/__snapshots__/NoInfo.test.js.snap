// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NoInfo component as expected 1`] = `
<div>
  <div
    class="no-info"
  >
    <div
      class="heart-broken-wrapper"
    >
      <i
        class="fas fa-heart-broken heart-broken"
      >
        <span
          class="sr-only"
        >
          Heart broken emoji
        </span>
      </i>
    </div>
    <h1
      class="no-info-heading"
    >
      Unfortunately, information about this stock is not available.
    </h1>
    <h1
      class="no-info-heading"
    >
      Please
       
      <a
        href="/"
      >
        search
      </a>
       
      for another stock.
    </h1>
  </div>
</div>
`;
